uint8 stream_type       # 0:录像流 1:主码流 2:子码流
uint8 encoding_type     # 1:H264 2:H265
uint16 resolution_width
uint16 resolution_height
uint16 bitrate_kbps
float32 frame_rate
